indico/modules/rb_new/client/js/common/rooms/RoomEditModal.jsx

```diff
@@ -295,30 +298,42 @@ class RoomEditModal extends React.Component {
         return <SpriteImage pos={position} />;
     };
 
-    removeAttribute = (attribute) => {
-        const {roomAttributes} = this.state;
-        this.setState({roomAttributes: _.omit(roomAttributes, attribute)});
-    };
-
-
-    renderAttributes = () => {
-        const {attributes, roomAttributes} = this.state;
+    renderAttributes = (content) => {
+        const {attributes} = this.state;
         if (!attributes) {
             return;
         }
-        return attributes.map(attribute => (
-            <>
-                {_.has(roomAttributes, attribute.name) && (
-                    <Field name={attribute.name}
-                           component={ReduxFormField}
-                           label={attribute.name}
-                           as={Input}
-                           isEqual={_.isEqual}
-                           icon={{name: 'trash', color: 'red', link: true, onClick: () => this.removeAttribute(attribute.name)}} />
-                )}
-            </>
-
-        ));
+        return (
+            <FieldArray name="attributes">
+                {({fields}) => {
+                    const options = this.generateAttributeOptions(attributes, fields.value);
+                    return (
+                        <div>
+                            <Dropdown button
+                                    text={content.placeholder}
+                                    className="icon"
+                                    floating
+                                    labeled
+                                    icon="add"
+                                    options={options}
+                                    search
+                                    disabled={options.length === 0}
+                                    selectOnBlur={false}
+                                    onChange={(event, values) => fields.push({title: values.value, value: null})} />
+                            {fields.map((attribute, index) => (
+                                <div key={attribute}>
+                                    <Field label={fields.value[index].title}
+                                        name={`${attribute}.value`}
+                                        component={ReduxFormField}
+                                        as={Input}
+                                        isEqual={_.isEqual}
+                                        icon={{name: 'trash', color: 'red', link: true, onClick: () => fields.remove(index)}} />
```

Comment:
if this creates a SUI Button you can  just set the `negative` prop to true instead of specifying a color